app/models/dialog_field_serializer.rb

```diff
@@ -12,9 +12,16 @@ def initialize(resource_action_serializer = ResourceActionSerializer.new)
   def serialize(dialog_field, all_attributes = false)
     serialized_resource_action = @resource_action_serializer.serialize(dialog_field.resource_action)
 
+    trigger_associations =
+      { dialog_field.name.to_s => dialog_field.dialog_field_trigger_associations.map { |field| DialogField.find(field.id).name } }
```

Comment:
I don't think we need to hit the database for the associated fields, you should just be able to call `field.name`